docs/ports.md

```diff
@@ -26,17 +26,24 @@
 | Koto                 | 9051                    | 9151                  | 8051             | 38351                       |
 | Bellcoin             | 9052                    | 9152                  | 8052             | 38352                       |
 | NULS                 | 9053                    | 9153                  | 8053             | 38353                       |
+| Bitcore              | 9054                    | 9154                  | 8054             | 38354                       |
 | Viacoin              | 9055                    | 9155                  | 8055             | 38355                       |
 | VIPSTARCOIN          | 9056                    | 9156                  | 8056             | 38356                       |
 | MonetaryUnit         | 9057                    | 9157                  | 8057             | 38357                       |
 | ZelCash              | 9058                    | 9158                  | 8058             | 38358                       |
 | Ravencoin            | 9059                    | 9159                  | 8059             | 38359                       |
 | Ritocoin             | 9060                    | 9160                  | 8060             | 38360                       |
 | Decred               | 9061                    | 9161                  | 8061             | 38361                       |
+| SnowGem              | 9062                    | 9162                  | 8062             | 38362                       |
 | Flo                  | 9066                    | 9166                  | 8066             | 38366                       |
 | Polis                | 9067                    | 9167                  | 8067             | 38367                       |
 | Qtum                 | 9088                    | 9188                  | 8088             | 38388                       |
 | Divi Project         | 9089                    | 9189                  | 8089             | 38389                       |
+| CPUchain             | 9090                    | 9190                  | 8090             | 38390                       |
+| DeepOnion            | 9091                    | 9191                  | 8091             | 38391                       |
+| Unobtanium           | 9092                    | 9192                  | 65535            | 38392                       |
+| Omotenashicoin       | 9094                    | 9194                  | 8094             | 38394                       |
+| BitZeny              | 9095                    | 9195                  | 8095             | 38395                       |
 | Sugarchain           | 9090                    | 9190                  | 8090             | 38390                       |
```

Comment:
Sugarchain's port and CPUchain's port are duplicated. I think we need to fix that.
I know that this code is only used for Sugarchain, but just in case.
Do you have a good idea about that?